Sources/FocusEntity/FocusEntity+Alignment.swift

```diff
@@ -160,9 +165,20 @@ extension FocusEntity {
     return results.first(where: { $0.target == .estimatedPlane })
   }
 
-  internal func performAlignmentAnimation(to newOrientation: simd_quatf) {
-    orientation = newOrientation
-  }
+    ///Uses interpolation between orientations to create a smooth `easeOut` orientation adjustment animation.
+      internal func performAlignmentAnimation(to newOrientation: simd_quatf) {
+        //interpolate between current and target orientations
+        orientation = simd_slerp(orientation, newOrientation, 0.15)
+        let testVector : simd_float3 = [0.7,0.7,0.7]
+        let point1 = orientation.act(testVector)
+        let point2 = newOrientation.act(testVector)
+        let distanceBetweenVectors = simd_distance(point1, point2)
+        //Stop interpolating when the rotations are close enough to each other.
+        if distanceBetweenVectors < 0.03 {
+            //Stop calling this function for horiztonal and vertical surfaces.
+            isChangingAlignment = false
+        }
```

Comment:
Hey Max! Great suggestion. I'm glad you found a way to compute this without calling acos(). The reason I do not want to use a vector along an axis is because one type of rotation (yaw, pitch, roll) will not affect the resulting vector. i.e. [0,0,1] is along the Z-Axis, and therefore roll-rotations will not change the vector resulting from .act().
For example, one of these quaternions represents a rotation of 180°  about the Z-Axis, and the other approximately 90° about the Z-Axis, but the dot product of the vectors resulting from .act() is 1.0.
```
let orientation = simd_quatf(angle: .pi, axis: [0,0,1])
let newOrientation = simd_quatf(angle: 1.57, axis: [0,0,1])

let testVector: simd_float3 = [0, 0, 1]
let point1 = orientation.act(testVector)
let point2 = newOrientation.act(testVector)

print(simd_dot(point1, point2))
//Prints 1.0
```

Maybe we could combine our solutions and use a testVector that is equivalent on all three components, and use simd_dot() instead of acos().

Reply:
oh of course, didn't consider the rotation about the z axis. I was thinking of changing to the Y axis instead, but that wouldn't be good enough either as it rotates about that when moving from two vertical walls.
The most important thing is using a normalised vector so that the dot product still gives cos of the angle, so using this as the test vector would works too while allowing rotations in x y and z:
`let testVector = simd_normalize(simd_float3(1,1,1))`
or [0.577, 0.577, 0.577], but I don't think that level of optimising is necessary